src/ansys/mapdl/core/launcher.py

```diff
@@ -1358,7 +1361,8 @@ def launch_mapdl(
         LOG.debug(
             "Because ``PYMAPDL_IP is not None, an attempt is made to connect to a remote session. ('START_INSTANCE' is set to False.`)"
         )
-        start_instance = False
+        if not ON_WSL:
```

Comment:
Worth considering adding an info message when this is true? Or even debug. Just in case

Reply:
Very good point!

Reply:
Done, but in the other side of the `if`.